templates/sensu-redis.json.j2

```diff
@@ -1,6 +1,14 @@
 {
   "redis": {
+{% if redis_password %}
+    "password": "{{ redis_password }}",
+{% endif %}
+{% if redis_sentinels %}
+    "sentinels": {{ redis_sentinels | to_nice_json(indent=4) }},
```

Comment:
Same question here, this would require everyone to update to ansible 2.2.X, so I have to ask why it's necessary to specify the indentation.